atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/TimestampChecker.java

```diff
@@ -15,17 +15,28 @@
  */
 package com.palantir.atlasdb.jepsen;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 
 import com.google.common.collect.ImmutableMap;
 import com.palantir.atlasdb.jepsen.events.Event;
+import com.palantir.atlasdb.jepsen.events.EventVisitor;
 
 import clojure.lang.Keyword;
 
 public final class TimestampChecker {
-    private TimestampChecker() {
+
+    List<EventVisitor> checkers;
+
+    public TimestampChecker(EventVisitor... checkers) {
+        this.checkers = Arrays.asList(checkers);
+    }
+
+    public static TimestampChecker createWithStandardCheckers() {
```

Comment:
The motivation was to minimise the amount of Clojure code, and also have `createWithStandardCheckers` be used by both Jepsen and by our integration test, so that they definitely stay in sync.